public/docs/ts/latest/guide/forms.jade

```diff
@@ -15,141 +15,130 @@ include ../_util-fns
   **two-way data binding, change tracking, validation, and error handling**
   ... which we shall cover in this guide on Angular forms.
 
-  We will build a simple form from scratch, one step at a time. Along the way we'll learn how to
+  We will build a simple form from scratch, one step at a time. Along the way we'll learn how to:
 
-  - build an Angular form with a component and template
-
-  - two-way data bind with `[(ngModel)]` syntax for reading and writing values to input controls
-
-  - track the change state and validity of form controls using `ngModel` in combination with a form
-
-  - provide strong visual feedback using special CSS classes that track the state of the controls
-
-  - display validation errors to users and enable/disable form controls
-
-  - use [template reference variables](./template-syntax.html#ref-vars) for sharing information among HTML elements
+  - Build an Angular form with a component and template
```

Comment:
According to the guidelines, we're supposed to put a period at the end of each list item, however, I have seen that not applied to TOCs, which I consider this to be. However, you'd be _within_ the guidelines by having periods on each since it isn't labeled explicitly as a TOC. I'd probably put them but considering the context, it's arguable not to have them.